crates/astria-composer/src/composer.rs

```diff
@@ -90,3 +165,45 @@ fn report_exit(task_name: &str, outcome: Result<eyre::Result<()>, JoinError>) {
         }
     }
 }
+
+#[async_trait::async_trait]
+impl ComposerService for ExecutorHandle {
```

Comment:
would prefer for all of `ExecutorHandle` to go in its own file/module, and also maybe name it `ComposerServer` or something like that?

Reply:
i will move the code to its own file/module.

Reply:
Looks like buf complains when I call it `ComposerServer` vs `ComposerService`. Plus we seem to be using the `Service` suffix for all our gRPC servers(https://github.com/astriaorg/astria/blob/72645d41c63515766528fba5fe616480553dcace/proto/executionapis/astria/execution/v1alpha2/execution.proto#L103). Should we just stuck to that?

Reply:
sorry, I meant to rename `ExecutorHandle` not `ComposerService`, but it's fine to leave as is